src/components/shared/MarkerContextMenu.js

```diff
@@ -269,26 +270,39 @@ class MarkerContextMenuImpl extends PureComponent<Props> {
           disabled={this._isZeroDurationMarker(marker)}
         >
           <span className="react-contextmenu-icon markerContextMenuIconSetSelectionFromMarker" />
-          Set selection from marker’s duration
+          <Localized id="MarkerContextMenu--set-selection-from-duration">
+            Set selection from marker’s duration
+          </Localized>
         </MenuItem>
         <div className="react-contextmenu-separator" />
         {this._isZeroDurationMarker(marker) ? (
           <>
             <MenuItem onClick={this.setStartRangeFromMarkerStart}>
               <span className="react-contextmenu-icon markerContextMenuIconStartSelectionHere" />
-              Start selection here
+              <Localized id="MarkerContextMenu--start-selection-here">
+                Start selection here
+              </Localized>
             </MenuItem>
             <MenuItem onClick={this.setEndRangeFromMarkerEnd}>
               <span className="react-contextmenu-icon markerContextMenuIconEndSelectionHere" />
-              End selection here
+              <Localized id="MarkerContextMenu--end-selection-here">
+                End selection here
+              </Localized>
             </MenuItem>
           </>
         ) : (
           <>
             <MenuItem onClick={this.setStartRangeFromMarkerStart}>
               <span className="react-contextmenu-icon markerContextMenuIconStartSelectionAtMarkerStart" />
               <div className="react-contextmenu-item-content">
-                Start selection at marker’s <strong>start</strong>
+                <Localized
+                  id="MarkerContextMenu--start-selection-at-marker-start"
+                  elems={{ strong: <strong /> }}
+                >
+                  <>
+                    Start selection at marker’s <strong>start</strong>
+                  </>
+                </Localized>
```

Comment:
optional: instead of doing
```js
<div>
  <Localized>
    <>
      text
    </>
  </Localized>
</div>
```
you can do:
```js
<Localized>
  <div>
    text
  </div>
</Localized>
```
This is also more like the other elements, where `<Localized>` is right after the `<span>` for the icon.

Reply:
Same remark for all similar cases below.